src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -454,6 +454,31 @@ def closeBlockContainer(self, name, scope='cms'):
             self.logger.error("Exception closing container/block: %s. Error: %s", name, str(ex))
         return response
 
+    def moveReplicationRule(self, ruleId, rseExpression, account):
+        """
+        Perform move operation for provided rule id and rse expression
+        :param ruleId: rule id
+        :param rseExpression: rse expression
+        :param account: rucio quota account
+        :param kwargs: positional arguments for rucio API
```

Comment:
Did you forget kwargs in the signature or in the docstring?

Reply:
ok, fixed docstring.